src/ui/login/Permission.svelte

```diff
@@ -39,9 +39,10 @@
     let accountNotFound: boolean = false
     let permissions: PermissionLevel[] | undefined
 
+    $: publicKey = walletPlugin.metadata.publicKey
```

Comment:
If this is now something Svelte watches (with the `$` syntax), can we actually overwrite it without any issues down on line 48?

Reply:
Worked fine, but turns out we don't need reactivity here like I thought.